mediaire_toolbox/queue/tasks.py

```diff
@@ -91,6 +110,8 @@ def create_child(self, tag=None):
         child_task = deepcopy(self)
         child_task.tag = tag
         child_task.update_timestamp = int(time.time())
```

Comment:
@pereferrera @kuuurt 

In `__init__`'s documentation, `timestamp` was supposed to be of `float` type (see remark above), but is assigned integer value here. Either one is wrong or both types are valid and the type should be `Union[float, int]`

Actually [PEP484 introduces a shortcut](https://www.python.org/dev/peps/pep-0484/#the-numeric-tower), but honestly I don't think this is what is intended here. 
> Rather than requiring that users write import numbers and then use numbers.Float etc., this PEP proposes a straightforward shortcut that is almost as effective: when an argument is annotated as having type float, an argument of type int is acceptable; similar, for an argument annotated as having type complex, arguments of type float or int are acceptable. This does not handle classes implementing the corresponding ABCs or the fractions.Fraction class, but we believe those use cases are exceedingly rare.
We should make the types explicit/